src/dotnet/HQ.Server/Services/TimeEntryServiceV1.cs

```diff
@@ -754,6 +754,24 @@ public async Task BackgroundSendTimeSubmissionReminderEmail(Period period, Cance
             }
         }
 
+        public async Task BackgroundSendRejectedTimeSubmissionReminderEmail(Period period, CancellationToken ct)
+        {
+            var startDate = DateOnly.FromDateTime(DateTime.UtcNow).GetPeriodStartDate(period);
+            var endDate = DateOnly.FromDateTime(DateTime.UtcNow).GetPeriodEndDate(period);
+            var times = _context.Times.Where(t => t.Date >= startDate && t.Date <= endDate);
+            var rejectedTimes = times.Where(t => t.Status != TimeStatus.Rejected);
+
+            var staffToNotify = await _context.Staff
+                .AsNoTracking()
+                .Where(t => t.EndDate == null && times.Where(x => x.StaffId == t.Id).Count() == 0 || rejectedTimes.Where(x => x.StaffId == t.Id).Count() > 0)
```

Comment:
We don't need the check for 0 hours `times.Where(x => x.StaffId == t.Id).Count() == 0`, we only care if there are rejected times.